client/my-sites/domains/domain-management/dns/dns-add-new.jsx

```diff
@@ -208,6 +253,9 @@ class DnsAddNew extends React.Component {
 		const selectedType = this.dnsRecords.find( ( record ) =>
 			record.types.includes( this.state.type )
 		);
+		const buttonLabel = recordToEdit
+			? translate( 'Update DNS record' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Update your DNS records' )` *ES Score: 7*